dist/hora.js

```diff
@@ -14,7 +14,13 @@ define([
         var $doc = $(document);
 
         var Hora = {
-            isDebug: false
+            isDebug: false,
+            event: {
```

Comment:
If we're just using this locally (a user wouldn't need to call Hora.event.accordiong), we don't need to bind this to Hora, and could possibly just use a local variable `eventTitles`